sdk/src/rent.rs

```diff
@@ -26,12 +28,16 @@ pub const DEFAULT_EXEMPTION_THRESHOLD: f64 = 2.0;
 /// default amount of rent to burn, as a fraction of std::u8::MAX
 pub const DEFAULT_BURN_PERCENT: u8 = ((50usize * std::u8::MAX as usize) / 100usize) as u8;
 
+/// default base rent
+pub const DEFAULT_BASE_RENT_PER_YEAR: u64 = 0;
```

Comment:
What if we expressed this in terms of DEFAULT_LAMPORTS_PER_BYTE_YEAR?  Say the base account overhead is whatever the current size of the Account structure currently is `* DEFAULT_LAMPORTS_PER_BYTE_YEAR`.   That'll still be zero currently so no functional change to this PR

Reply:
@mvines We decided to detach this from current size of the Account structure right? Otherwise, we need to do hard fork in case size changes in future.

Reply:
Yes.  I was just suggesting we use the current size of the Account structure as the value we hard code here (rather than randomly picking a value)

Reply:
@mvines hmm. This could work out. But, do you think hard coding it is a good idea compared to keeping it as default, and allowing us to overwrite it in future?

Reply:
I just mean this:
```diff
-pub const DEFAULT_BASE_RENT_PER_YEAR: u64 = 0;
+pub const DEFAULT_BASE_RENT_PER_YEAR: u64 = 128 * DEFAULT_LAMPORTS_PER_BYTE_YEAR;
```
128 seems is approximately the current Account size.

Reply:
maybe that's too much for a base rent 🤷‍♂

Reply:
ahh, I misunderstood. Cool, this works.

Reply:
@mvines done.